garden_ai/templates/pipeline

```diff
@@ -64,6 +64,5 @@ REQUIREMENTS_FILE = None  # to specify additional dependencies, replace `None`
     description="{{ pipeline.description }}",
     version="{{ pipeline.version }}",
     year={{ pipeline.year }},
-    tags={{ pipeline.tags }},
-    uuid="{{ pipeline.uuid }}",  # WARNING: DO NOT EDIT UUID
+    tags={{ pipeline.tags }}
```

Comment:
I feel like we may have touched on this synchronously with Will, but I _think_ we'll need the DOI as part of the scaffold (probably with the same warning comment) otherwise if registers a pipeline; edits it; re-registers the pipeline, we'll have minted (and permanently published w/ datacite) two different dois for the same entity.